opentracing/defaulttracer.go

```diff
@@ -0,0 +1,57 @@
+package opentracing
+
+var (
+	defaultOpenTracer OpenTracer = noopOpenTracer{noopTraceContextSource{}}
```

Comment:
this is a great point -- disregard previous response (if you saw it via email), I misremembered what the code actually did here. will fix.